### ISSUE TITLE
Exception calling client.destroy(inf_id)

### ISSUE THREAD

**Issue by sebastian-luna-valero** [grycap/im-client#108]
Hi,

I am testing the im-client's python library, following the example in the README:
https://github.com/grycap/im-client/blob/1dd4ee173074465bad24778732bead55b2bf73ef/README.md#15-python-client

And I am getting:
```bash
  File "/tmp/tmp.xavoPdXogE/conda-install/envs/poetry/lib/python3.13/site-packages/imclient/imclient.py", line 841, in destroy
    if self.options.force:
       ^^^^^^^^^^^^^^^^^^
AttributeError: 'Values' object has no attribute 'force'
```

Pointing to:
https://github.com/grycap/im-client/blob/1dd4ee173074465bad24778732bead55b2bf73ef/imclient/imclient.py#L841

Could you please have a look?

Thanks!

**Comment by micafer** [comment]
Could you try cloning the repo now?

**Comment by sebastian-luna-valero** [comment]
Many thanks!

I am now getting:

```bash
/tmp/tmp.xavoPdXogE/conda-install/envs/poetry/lib/python3.13/site-packages/imclient/imclient.py:418: SyntaxWarning: invalid escape sequence '\s'
  success, infras = self._list_infras(flt=".*description\s*.*\s*(\s*name\s*=\s*'%s'.*).*" % inf_id)
```

but seems to work.

**Comment by sebastian-luna-valero** [comment]
It is, thank you very much!

**Comment by sebastian-luna-valero** [comment]
Hi,

I see the same warning in a fresh install of `im-client` version `1.8.1`:
```bash
/tmp/tmp.aBpxexO8kQ/conda-install/envs/poetry/lib/python3.12/site-packages/imclient/imclient.py:418: SyntaxWarning: invalid escape sequence '\s'
  success, infras = self._list_infras(flt=".*description\s*.*\s*(\s*name\s*=\s*'%s'.*).*" % inf_id)
```
Is this expected?

Thanks!

**Comment by sebastian-luna-valero** [comment]
Hi,

Coming back to this, it looks like these errors were fixed in `1.8.1`:
https://github.com/grycap/im-client/milestone/14

But `1.8.1` was never released? See
* https://github.com/grycap/im-client/releases
* https://pypi.org/project/IM-client/#history

Thanks,
Sebastian
